frontend/src/services/user-follower-api/user-follower-api.service.ts

```diff
@@ -0,0 +1,64 @@
+import {
+  ApiPath,
+  ContentType,
+  HttpMethod,
+} from 'common/enums/enums';
+import { Http } from 'services/http/http.service';
+import { UserFollower, UserFollowerPayload } from 'common/types/types';
+
+type Constructor = {
+  http: Http;
+  apiPrefix: string;
+};
+
+class UserFollowerApi {
+  #http: Http;
+  #apiPrefix: string;
+
+  constructor({ http, apiPrefix }: Constructor) {
+    this.#http = http;
+    this.#apiPrefix = apiPrefix;
+  }
+
+  public getCountByUserId(userId: number): Promise<number> {
+    return this.#http.load(
+      `${this.#apiPrefix}${ApiPath.USER_FOLLOWERS}/${userId}`,
+      {
+        method: HttpMethod.GET,
+      },
+    );
+  }
+
+  public isFollowed(payload: UserFollowerPayload): Promise<UserFollower> {
```

Comment:
```suggestion
  public checkIsFollowed(payload: UserFollowerPayload): Promise<UserFollower> {
```
functions with this name should return boolean